### PR TITLE
Update to quasar-80.0.0

### PR DESCRIPTION
**Pull request by wemrysi**

### CODE REVIEW

File: datasource/src/main/scala/quasar/physical/s3/impl/evaluate.scala [hunk 1]
```diff
@@ -21,74 +21,38 @@ import slamdata.Predef._
...
 
-    sign(request) >>= { req =>
-      streamRequest[F, R](client, req, file) { resp =>
-        resp.body.chunks.map(_.toByteBuffer).through(parse(jsonParsing))
-      }.map(_.handleErrorWith {
-        case ParseException(message, _, _, _) =>
-          Stream.eval(MR.raiseError(parseError(file, jsonParsing, message)))
-      })
-    }
+    sign(request) >>= (streamRequest[F, Byte](client, _, file)(_.body))
```
[the remark this answers is not in the document]
Comment (wemrysi) [reply datasource/src/main/scala/quasar/physical/s3/impl/evaluate.scala]: Yes, but this has to be handled in quasar now.

**Comment by wemrysi** [comment]
@jsantos17 QA also identified performance regressions, as of the fs2 upgrade. Did you happen to do any performance testing on just the fs2 changes? I'd be surprised if any of the changes in this PR had any performance impact.

File: datasource/src/main/scala/quasar/physical/s3/impl/evaluate.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsantos17) [reply datasource/src/main/scala/quasar/physical/s3/impl/evaluate.scala]: We still want a good user-facing error message for this case, right? Happy to open another ticket and address it separately.

[the remark this answers is not in the document]
Comment (wemrysi) [reply datasource/src/main/scala/quasar/physical/s3/impl/evaluate.scala]: Yes, I'm just saying there's nothing we can do about it here.

**Comment by jsantos17** [comment]
@wemrysi yes, I ran the same test with this PR and also with `79.1.1` (in both slamdata-backend and this repo). `79.1.1` doesn't have this problem.

**Comment by jsantos17** [comment]
@wemrysi I was able to reproduce this with the local connector. So it's not limited to this PR.

**Comment by wemrysi** [comment]
@jsantos17 quasar-80.0.0 introduces tectonic and parser pushdown, so root-level shifts on datasets will get pushed down into the parser and speed things up quite a bit, which is probably what you're seeing.

Seems like something in the tectonic pipeline (parser, plate or plumbing) is slowing things up overall, though (/cc @djspiewak). However, QA also noted some fairly severe regressions with just the fs2 upgrade.

**Comment by djspiewak** [comment]
@jsantos17 @wemrysi It definitely sounds like something is up with tectonic. I haven't benchmarked the intermediate plates at all, though I did benchmark `SlicePlate` and it is is vastly faster than the old `fromQDataStream` mechanism. I should probably prioritize getting on that next week.

**Pull request merged**